### ISSUE TITLE
Provide code action to turn off a hlint warning in a project

### ISSUE THREAD

**Issue by alanz** [haskell/haskell-ide-engine#955]
When a hlint warning is provided, e.g. "use camelCase", you do not always want to apply it in the project.

Provide a code action to add it to the file `.hlint.yaml` in the project root, creating that file if it does not exist.

```yaml
- ignore: {name: "Use camelCase"}
```
If possible, provide another one for a single override at this location. If hlint provides such a feature.

**Comment by jean-lopes** [comment]
Hey, can I help with this issue?

**Comment by alanz** [comment]
Yes indeed.  You basically need to modify the `codeActionProvider` method in the `ApplyRefact` plugin, to keep an eye out for hints that look like that, and expose a command that will then write it to that file, creating it if it does not exist.

Shout if you need guidance.

**Comment by anpandey** [comment]
Hi, if nobody's working on this issue, could I try working on it?
